playwright-e2e/pages/page-base.ts

```diff
@@ -267,8 +364,12 @@ export default abstract class PageBase implements PageInterface {
    * <br> Question: How old are you?
    * <br> Type: Input
    */
-  age(): Input {
-    return new Input(this.page, { ddpTestID: 'answer:AGE' });
+  age(opts?: { testId: string}): Input {
+    if (!opts) {
+      return new Input(this.page, { label: 'Enter age' });
+    }
+    // 'answer:AGE'
+    return new Input(this.page, { ddpTestID: opts.testId });
```

Comment:
Make this function more generic. ddpTestID are different across studies but I don't want see different `age()` functions for each study.